app/views/transcripts/charts/_bubble_courses.html.haml

```diff
@@ -1,5 +1,13 @@
 #chart-bubble-courses.chart
 
+<!-- This style will hide bubblechart tooltips third elements that currently happens 
```

Comment:
Everything worked right, you just must put this style on `app/assets/stylesheets/application.css.scss` file. And since the tag qualifying `div#chart-bubble-course` is not required, you can do something like:

```
#chart-bubble-courses > div > div:last-child li:nth-child(3) {
  display: none;
}
```

Triggering the same effect and being more legible!